### PR TITLE
Add website credits for licenses used 

### LINKED ISSUE #95: Incorporate credit for products with free licenses
We are currently using several products for which we've obtained free licenses. Credit for this usage needs to be incorporated into worldwind.arc.nasa.gov. 

Licenses that need to be incorporated include: 
- IntelliJ IDEA
- WebStorm 
- Install4J

Credit for these licenses will be given on the "Overview" Pages for WWJava, Web WW, and WWSK, respectively.

### PR DESCRIPTION
**Pull request by eirizarry**
This PR satisfies the requirements giving due credit to the open source licenses we use. 

The following changes have been made: 
- Add .svg files of ESA logo, WebStorm logo, and IntelliJ IDEA logo
- Add logo and links which lead to each of these sites
- Move the ESA logo to give it more visibility within Web WorldWind's pages

Closes #95

### CODE REVIEW

**Review by pdavidc — CHANGES_REQUESTED** [review]
Very nice addition of our IDE credits. I especially like the change to WebWorldWind attributing ESA.

One adjustment is needed: We use Android Studio for WorldWindAndroid development. Since Android Studio is free, it's up to us whether we mention it.

File: content/android/_index.md [hunk 1]
```diff
@@ -20,3 +20,5 @@ draft: false
 - [GitHub Issues](https://github.com/NASAWorldWind/WorldWindAndroid/issues/) lists WorldWind bugs, issues, and
         milestones
 - [Forum](https://forum.worldwindcentral.com) provides help from the WorldWind community
+
+---
```
Comment (pdavidc) [inline content/android/_index.md]: Did you mean to leave the trailing horizontal line?

Comment (eirizarry) [reply content/android/_index.md]: My apologies, I'll remove it!

**Comment by pdavidc** [comment]
Looks great. I really like the SVG images.

**Review by pdavidc — APPROVED** [review]

**Pull request merged**